src/components/atoms/Snackbar.tsx

```diff
@@ -3,18 +3,19 @@ import MuiSnackbar from "@mui/material/Snackbar";
 import { useAppDispatch, useAppSelector } from "../../store/store";
 import { hideSnackbar, selectSnackbarState } from "../../slices/snackbarSlice";
 import { useTranslation } from "react-i18next";
-import { ErrorMessage } from "../../constants/ErrorMessage";
-import { getErrorKey } from "../../utils/authUtils";
+import { SnackbarMessage } from "../../constants/ErrorMessage";
+import { getMessageKey } from "../../utils/authUtils";
 import { Alert } from "@mui/material";
 
 const Snackbar = () => {
-  const { isOpen, message, autoHideDuration, severity } = useAppSelector(selectSnackbarState);
+  const { isOpen, message, autoHideDuration, severity } =
+    useAppSelector(selectSnackbarState);
   const dispatch = useAppDispatch();
   const messageKey = message ? message : "";
   const { t } = useTranslation();
 
-  const errorMessage =
-    ErrorMessage[getErrorKey(messageKey) as keyof typeof ErrorMessage];
+  const snackbarMessage =
```

Comment:
look if Firebase has a type with error values exported so you can use it as a type 
also I would suggest some default error if somehow the message doesn't match any of those that you have